wire-ios-data-model/WireDataModel.xcodeproj/project.pbxproj

```diff
@@ -3215,6 +3222,24 @@
 			shellPath = /bin/sh;
 			shellScript = "# Adds support for Apple Silicon brew directory\nexport PATH=\"$PATH:/opt/homebrew/bin\"\n \nif which swiftlint >/dev/null; then\n  swiftlint\nelse\n  echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi\n";
 		};
+		EE47346529A377D100E6C04E /* Run Sourcery */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = "Run Sourcery";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "# Adds support for Apple Silicon brew directory\nexport PATH=\"$PATH:/opt/homebrew/bin\"\n \nif which sourcery >/dev/null; then\n  sourcery --config ./sourcery/config.yml\nelse\n  echo \"warning: Sourcery not installed, download from https://github.com/krzysztofzablocki/Sourcery\"\nfi\n\n";
```

Comment:
does this mean CI need to have it install too?
Can we just add it as a script we run when needed, and checked the generated code into git?

Reply:
➕ for keeping the generated code in the git repository

Reply:
@netbe no the CI doesn't need this, if the soucery tool isn't installed on the local machine the script doesn't do anything. The generated code is checked into git already, so you don't need to run the script to build the project. You can simply run sourcery from the command line, but adding it here means you only need to build to get the mocks updated.